jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/spec/filter/interceptor/JAXRSClientIT.java

```diff
@@ -620,6 +621,110 @@ public void fileWriterClientInterceptorTest() throws Fault {
     logMsg("JAXRS called registered writer interceptor for file provider");
   }
 
+  // ------------------------- Path -----------------------------------
+
+  /*
+   * @testName: pathReaderContainerInterceptorTest
+   * 
+   * @assertion_ids: JAXRS:SPEC:84;
+   * 
+   * @test_Strategy: JAX-RS implementations are REQUIRED to call registered
+   * interceptors when mapping representations to Java types and vice versa.
+   */
+  @Test
+  @Tag("xml_binding")
```

Comment:
Question:  All of these tests (including what was added here) have this `@Tag`.  Do they really all require `xml_binding`?   If not then this may be a problem (even for Jakarta Rest 4.0) since xml_binding is no longer required by the Platform, so none of these tests will run assuming tests tagged this way will be excluded.  This applies to all new tests in this PR.

Reply:
+1, given Jakarta XML Binding is no longer required, implementations won't need to run these effectively allowing implementations to pass without even implementing the feature.

Reply:
I actually did a 1:1 copy from the "file" originals, just replacing "file" by "path". So I assume that why you mention was already wrong before. I hence kindly ask to separate that discussion from this PR, and address that issue in a separate PR.

Reply:
I've written [Issue 1277](https://github.com/jakartaee/rest/issues/1277) to document this.   However, I don't see the point of adding additional tests that will not run.  I understand that you may not have time to evaluate/address all of the tests in this class, but any new that you add should not have that dependency.

Reply:
So what is the correct solution?
* Simply removing `@Tag("xml_binding")` from the added test?
* Not adding the test at all?

Reply:
Since it was just a copy/paste of a different test, hopefully simply removing that tag won't prevent it from running.   Otherwise the test would need to be modified to not require that tag.  I don't think "Not adding the test at all" is an appropriate option.

Reply:
I checked these tags and need to say that I would prefer to keep them as part of *this* MR as we should not mix two otherwise independent issues. The point is that *this* issue is solely a 1:1 copy of existing code, and there is already a plan to fix that existing code, so *then* it should be easy to fix these copies in one shot. The problem with fixing *this* MR upfront simply is that they are embedded in a larger framework, and it is *that larger framework* that depends on JAXB. So a fix of the test copies is *impossible* without fixing the original tests in the same commit. As there already are plans to fix those original tests eventually, it makes not much sense to mix things here and make my life in *this* MR harder than essentially needed.

Having said that, please let's fix the tags *later* so we can simply and easily proceed *here*. Thanks.

Reply:
I see your point Markus.  As long as [Issue 1277](https://github.com/jakartaee/rest/issues/1277) is addressed prior to the next release.   Can you confirm that the new TCK tests you've added have run successfully (since they currently would be skipped as part of a Jakarta Rest 4.0 TCK execution correct?)

Reply:
I'm not sure we should be adding new TCK tests that don't run. It kind of defeats the purpose of a test. While I can understand that's how the old tests work, there's not reason to make new tests work the same way. There could very easily be a new `jsonContent` instance variable created that these tests use. I see no reason for it to to be XML content. It could even be plain text.

There would be new interceptors needed too, but again IMO that's not really a big deal.

Reply:
@jim-krueger I cannot confirm that they run, as there is not yet any implementation that fulfils the new API.

Reply:
How do we procedd? Apparently nobody chimed in to fix #1277, so could we please merge *this* feature and solve #1277 *later*?